.github/CHANGELOG.md

```diff
@@ -62,6 +62,9 @@
 * `SX` and `C(SX)` gates are natively supported for all lightning devices.
   [(#731)](https://github.com/PennyLaneAI/pennylane-lightning/pull/731)
 
+* Programs transformed by `qml.defer_measurements` can be executed on `lightning.qubit`.
```

Comment:
This is the result of https://github.com/PennyLaneAI/pennylane/pull/6838. Technically, this PR adds support for `ctrl` and `adjoint` transforms, as well as tests for `defer_measurements`. Could you update the title of this PR accordingly?

Reply:
Done.